pages/user/auth/index.tsx

```diff
@@ -0,0 +1,131 @@
+import * as phase from "@/components/styles/user/first/style";
+import * as styles from "@/components/styles/user/third/style";
+import { Button, Checkbox, TextField } from "@mui/material";
```

Comment:
사용안하시는 컴포넌트(Checkbox, TextField) 는 지워주세요~!
```suggestion
import { Button } from "@mui/material";
```